dashboard/src/pages/Capability/Components/ShowComponent/index.tsx

```diff
@@ -73,7 +73,7 @@ export default ({ name, parameters }: ShowParameters) => {
             responsive: ['md'],
             render: (text, row) => [
```

Comment:
Should not return an array, should return a element

Reply:
fixed